VideoLocker/src/main/java/org/edx/mobile/player/PlayerController.java

```diff
@@ -539,12 +532,16 @@ public void updatePausePlay() {
         if (mRoot == null || mPauseButton == null || mPlayer == null) {
             return;
         }
-
-        mPauseButton.setBackgroundColor(Color.TRANSPARENT);
         if (mPlayer.isPlaying()) {
-            mPauseButton.setImageResource(R.drawable.ic_pause_button_selector);
+            mPauseButton.setImageDrawable(new IconDrawable(getContext(), Iconify.IconValue.fa_pause)
+                    .colorRes(R.color.edx_grayscale_neutral_white_t));

```

Comment:
This should really have been set up as a `ToggleButton` but it looks like it doesn't support image toggling :frowning: so I guess it's fine for now. However, if we use this pattern at some other point, it might be worth it to create a custom widget for it (that also supports content description toggling).